vitrina/datasets/views.py

```diff
@@ -734,6 +735,8 @@ def get_form_class(self):
 
         if subclass.name == DCATResourceSubclass.SERVICE:
             return ServiceResourceForm
+        elif subclass.name == DCATResourceSubclass.INFORMATION_SYSTEM:
```

Comment:
Just nitpicking: `elif` is redundant:
```suggestion
        if subclass.name == DCATResourceSubclass.INFORMATION_SYSTEM:
```

Reply:
Fixed in 3ea040ee4611a47b23f6dc6472a957502c116590